drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/GitHubSource.java

```diff
@@ -116,12 +111,13 @@ public ExternalBinary getReleaseForVersion(String version) throws Exception {
                 break;
             }
         }
-        log.warning(
-            "There wasn't found any release for the version: " + version + " in the repository: " + projectUrl);
-        return null;
+        throw new IllegalArgumentException(
+            "There wasn't found any release for the version: " + version + " in the repository: " + projectUrl
```

Comment:
How about "No release matching version xxx has been found in the repository yy......" ?